docs/outlook/add-in-commands-for-outlook.md

```diff
@@ -52,23 +52,25 @@ In Outlook on the web, the add-in name is displayed in an overflow menu. If the
 
 ![Overflow menu displaying add-in command buttons.](../images/commands-overflow-menu-expand-web.png)
 
-## What UX shapes exist for add-in commands?
+## What are the types of add-in commands?
 
-The UX shape for an add-in command consists of a ribbon tab in the Office application that contains buttons that can perform various functions. Currently, three UI shapes are supported:
+The UI for an add-in command consists of a ribbon button or an item in a drop-down menu. There are two types of add-in commands, based on the kind of action that the command triggers.
 
-- A button that executes a JavaScript function
-- A button that launches a task pane
-- A button that shows a drop-down menu with one or more buttons of the other two types
+- **Task pane commands**: The button or menu item opens the add-in's task pane. You add this kind of add-in command with markup in the manifest. The "code behind" the command is provided by Office.
+- **Function commands**: The button or menu item runs any arbitrary JavaScript. The code almost always calls APIs in the Office JavaScript Library, but it doesn't have to. This type of add-in typically displays no UI other than the button or menu item itself. Note the following about function commands:
 
-### Executing a JavaScript function
+   - The function that is triggered can call the [displayDialogAsync](/javascript/api/office/office.ui?view=common-js&preserve-view=true#office-office-ui-displaydialogasync-member(1)) method to show a dialog, which is a good way to display an error, show progress, or prompt for input from the user.
```

Comment:
I'm don't think you want to lock to a specific moniker here. If the reader is already in Excel or Word, they should stay there, since this Common page is always present.
```suggestion
   - The function that is triggered can call the [displayDialogAsync](/javascript/api/office/office.ui#office-office-ui-displaydialogasync-member(1)) method to show a dialog, which is a good way to display an error, show progress, or prompt for input from the user.
```